homeassistant/components/tradfri/base_class.py

```diff
@@ -61,10 +61,13 @@ def device_info(self):
 
         return {
             "identifiers": {(TRADFRI_DOMAIN, self._device.id)},
-            "name": self._name,
             "manufacturer": info.manufacturer,
             "model": info.model_number,
+            "name": self._name,
```

Comment:
Are there any constants available for these keys or are they named on a per-integration-basis?